packages/beacon-node/test/spec/utils/specTestIterator.ts

```diff
@@ -65,9 +65,10 @@ export const defaultSkipOpts: SkipOpts = {
   skippedTestSuites: [
     /^capella\/light_client\/single_merkle_proof\/BeaconBlockBody.*/,
     /^deneb\/light_client\/single_merkle_proof\/BeaconBlockBody.*/,
+    /^electra\/light_client\/.*/,
   ],
   // TODO Electra: Review this test in the next spec test release
-  skippedTests: [/incorrect_not_enough_consolidation_churn_available/],
+  skippedTests: [/incorrect_not_enough_consolidation_churn_available/, /^deneb\/light_client\/sync\/.*electra_fork.*/],
```

Comment:
I believe `/incorrect_not_enough_consolidation_churn_available/` can be removed since it was fixed after alpha.3